modules/oe_subscriptions_anonymous/src/Form/AnonymousSubscribeForm.php

```diff
@@ -200,17 +195,11 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
    * {@inheritdoc}
    */
   protected function successfulAjaxSubmit(array $form, FormStateInterface $form_state) {
+    $form_state->disableRedirect(FALSE);
```

Comment:
At first this scared me, as core explicitly disables this. But I understand you are doing this to retrieve the redirect set in the standard submit callback.
Let's:

1. add a comment that this is our intention: so developers that don't know that redirects are disabled in ajax requests (like myself) don't need to check up the code.
2. Store the current status of redirect (`isRedirectDisabled`) in a variable.
3. Collect the redirect from the form state.
4. Restore the original value.

Reply:
Done